dockerfiles/start.sh

```diff
@@ -2,13 +2,9 @@
 
 if [[ "${DCAP_LIBRARY}" == "intel" ]]
 then
-    # rename the library installed by az-dcap-client
-    mv /usr/lib/libdcap_quoteprov.so /usr/lib/libdcap_quoteprov.so.azure
-    # create a link to the intel quote provider library
-    ln -s /usr/lib/x86_64-linux-gnu/dcap/libdcap_quoteprov.so.intel /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so
-    ln -s /usr/lib/x86_64-linux-gnu/dcap/libdcap_quoteprov.so.intel /usr/lib/x86_64-linux-gnu/libdcap_quoteprov.so.1
+    apt-get install -qq libsgx-dcap-default-qpl > /dev/null
```

Comment:
```suggestion
    apt-get install -qq libsgx-dcap-default-qpl > /dev/null 2>&1
```
Did some quick testing, if we want to silence all output we need to redirect stdout and stderr to /dev/null